### DISCUSSION TITLE
Docker error

### DISCUSSION THREAD

**Discussion by zsamiatt** [photoprism/photoprism#2148] [Q&A] [answered]
Hello,
I updated my docker file to new docker file. But latest docker file can not running.

init as root

image: amd64-prod, build 220302-bullseye

home: /photoprism

umask: "0002" (u=rwx,g=rwx,o=rx)

install-path: /opt/photoprism

running as uid 0

photoprism start

/entrypoint.sh: line 104: audit.sh: command not found

Can u help me?

**Comment by graciousgrey** [comment, marked as answer]
See: https://github.com/photoprism/photoprism/discussions/2120
